plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -251,7 +252,10 @@ public Object start(IApplicationContext context) {
         writeWorkspaceInfo();
 
         // Update splash. Do it AFTER platform startup because platform may initiate some splash shell interactions
-        updateSplashHandler();
```

Comment:
Please avoid copy-paste. Create function isMacOsSanoma.

Reply:
Method extracted.

Reply:
It is still here

Reply:
Moved to RuntimeUtils